exercises/atbash-cipher/example.ml

```diff
@@ -0,0 +1,57 @@
+let substitute = function
+  | 'a' .. 'z' as c ->
+    let offset = (Char.code c) - (Char.code 'a') in
+    let code = (Char.code 'z') - offset in
+    Char.chr code
+  | other -> other
+
+let is_encodable = function
+  | 'a' .. 'z' | '0' .. '9' -> true
+  | _ -> false
+
+let explode s =
```

Comment:
The signature of this function is not immediately obvious (at least to me!). A Haskell convention is to provide type signatures on top level functions in a module - that gives useful, and compiler checked, documentation. I'd favour the same in Ocaml code.